collective/js/throttledebounce/profiles/default_plone5/registry.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0"?>
+<registry>
+
+  <records prefix="plone.bundles/collective-js-throttledebounce-resources"
+           interface="Products.CMFPlone.interfaces.IBundleRegistry">
+    <value key="enabled">True</value>
+    <value key="depends">plone</value>
+    <value key="stub_js_modules">jquery</value>
+    <value key="compile">False</value>
+    <value key="jscompilation">++plone++collective.js.throttledebounce/throttledebounce-compiled.js</value>
```

Comment:
I'm not quite sure but I think this should be `++resource++collective...` according to the directory registry naming.

Reply:
👍 thanks